books/free-programming-books-ko.md

```diff
@@ -161,6 +161,7 @@
 
 ### Python
 
+* [금융 데이터 분석을 위한 파이썬 판다스](https://wikidocs.net/book/3488) - 조대표
```

Comment:
this book appears to be available from a number of sources, why did you pick this one ? the ads are intrusive.